infra/gp-cluster-logging-operator/templates/namespaces.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: ""
```

Comment:
entweder Patch "patch-add-label-to-namespace" löschen und das Label hier hinzufügen, oder die anderen Namespaces ebenfalls über Patches Labeln. 
Ersteres ist vermutlich die schönere Variante